README

```diff
@@ -1,2 +1,10 @@
-# Neuromod: xxxx subdataset
+# CNeuromod: [xxxx subdataset](https://github.com/courtois-neuromod/xxxxx)
 
+This BIDS dataset is part of the [Courtois-Neuromod dataset](https://docs.cneuromod.ca/).
+
+This subdataset contains functional MRI data collected with xxxxx task.
+
+It contains:
+- raw neuroimaging data in BIDS format
+- behavioral variables
+- physiological data
```

Comment:
Suggestion: add author and access information. 

This dataset was designed by XX, with input from XX, XX. The subdataset was collected  by XX, XX. Further improvements on data organization as well as quality control have been implemented by XX, XX. This subdataset also benefited from resources assembled by the entire [Courtois NeuroMod team](https://docs.cneuromod.ca/en/latest/AUTHORS.html).

Courtois NeuroMod data are freely shared with the scientific community to advance research at the interface of neuroscience and artificial intelligence. Access to to the data is based on a registered access model, see the [Courtois NeuroMod website](https://docs.cneuromod.ca/en/latest/ACCESS.html) for more information. Redistribution of data outside of the Courtois NeuroMod data bank is prohibited.